### ISSUE TITLE
searching by repertory

### ISSUE THREAD

**Issue by holfordm** [bodleian/medieval-mss#193]
## Detailed Description
As part of the future development of the medieval catalogue, it would be useful for users to search by repertory numbers for texts that are usually identified in this way (e.g. numbers in the Index of Middle English verse, Bibliotheca hagiographica latina for saints lives, or the repertory of Thorndike and Kibre for medical texts). For an example of how this might work, see http://pinakes.irht.cnrs.fr/recherche-generale.html, search box 'repetoire' and 'numero'.

Is this desirable, and would this be a suitable way to mark this information up (linking to an authority file of repertoria)

`<bibl type="repertory">
                        <title key="repertory_1">Thorndike-Kibre</title>
                        <citedRange unit="entry">471D</citedRange>
                     </bibl>`

**Comment by andrew-morrison** [comment]
So these are external catalogues, relating works to subjects? And the idea would be someone could find which Oxford manuscripts relate to, say, a particular saint, if they know the Bibliotheca hagiographica latina number?

**Comment by holfordm** [comment]
Yes, that's right. For some cases, like the Index of Middle English Verse (http://www.dimev.net/) this wouldn't add much value, because those resources already aim to have a complete list of manuscripts, but in other cases it would be useful, particularly for shortish texts where the entry number in a repertory is the usual/only identifier.

**Comment by andrew-morrison** [comment]
Would these new `bibl` elements go in the works authority file, or in the `msItem` elements in the manuscript TEI files?

**Comment by holfordm** [comment]
probably in both - for longer works, which are indexed in the works authority file, the identifiers would go there, for shorter works, which don't often get an authority entry, in the `msitem`.
